lib/_http_client.js

```diff
@@ -274,7 +259,7 @@ function ClientRequest(options, cb) {
     this._last = true;
     this.shouldKeepAlive = false;
     if (typeof options.createConnection === 'function') {
-      newSocket = options.createConnection(options, oncreate);
+      var newSocket = options.createConnection(options, oncreate);
```

Comment:
Good call! I left it as `var` as it had already been that on line 244 previously, but `const` is more appropriate here now.